options/m_config.h

```diff
@@ -201,6 +201,10 @@ int m_config_option_requires_param(struct m_config *config, bstr name);
 // Notify m_config_cache users that the option has (probably) changed its value.
 void m_config_notify_change_co(struct m_config *config,
                                struct m_config_option *co);
+// Like m_config_notify_change_co(), but automatically find the option by its
```

Comment:
In theory it could be moved before this commit, but it'd essentially do nothing. This is just a hack anyway, until we figure out how changing options (outside of the option/property system) should work.